.travis.yml

```diff
@@ -1,9 +1,12 @@
 sudo: false
 language: node_js
 node_js:
-  - "6"
   - "8"
 
+os:
+  - linux
+  - osx
```

Comment:
Please don't make two changes in one pull request. Adding `osx` to our build matrix should be done in a standalone pull request.

Reply:
The historic reason for this addition is that Travis was only running commit linter instead of our build, since there was no longer a matrix with https://github.com/strongloop/loopback-next/pull/945/commits/1e5bc766ff6e85a4ebb20a39c0b6b35826099294. TV and I added it to address that issue. I agree that is a change on its own, so I can create a PR for it.

Reply:
Wow, I did not realize a single Node.js target would break the matrix build. Thanks for explaining this to me. Your other PR looks good to me. Another option we can consider is to add Node.js 9 to the list of platforms we are testing.